test/integration/docker_opt_test.go

```diff
@@ -0,0 +1,56 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/minikube/test/integration/util"
+)
+
+func TestDockerOpt(t *testing.T) {
```

Comment:
My only comment here is that we might want to consider merging TestDockerOpt and TestDockerEnv so that we don't add another recreate cycle here.  Even those its technically less correct, we want to make sure that our integration tests run in a reasonable time frame.